packages/p2p/src/components/order-details/__tests__/order-details-wrapper.spec.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import { useStores } from 'Stores';
+import { isMobile } from '@deriv/shared';
+import OrderDetailsWrapper from '../order-details-wrapper';
+
+const MockOrderWrapper = props => {
+    return isMobile() ? (
+        <OrderDetailsWrapper {...props}>
+            <p>desktop test</p>
+        </OrderDetailsWrapper>
+    ) : (
+        <OrderDetailsWrapper {...props}>
+            <p>mobile test</p>
+        </OrderDetailsWrapper>
+    );
+};
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+}));
+
+const mockUseStores = () => {
+    useStores.mockImplementation(() => ({
+        order_store: { order_information: { should_show_order_footer: true } },
+        sendbird_store: { setShouldShowChatModal: false },
+    }));
+};
+
+beforeAll(() => mockUseStores());
+
+describe('<OrderDetailsWrapper />', () => {
+    it('Should render component on desktop version', async () => {
+        isMobile.mockReturnValue(true);
```

Comment:
```suggestion
        isMobile.mockReturnValue(false);
```
This should be false for setting the desktop version I presume. Same for the mobile version below, I think it was mistakenly flipped.

Reply:
I think the flow will be correct here, But the confusion is with  line 8 . Desktop element will render when "isMobile()" function returns true.